### PR TITLE
rosdep: Added libexiv2-dev

### PR DESCRIPTION
**Pull request by peci1**
Please add the following dependency to the rosdep database.

## Package name:

libexiv2-dev

## Package Upstream Source:

https://www.exiv2.org/

## Purpose of using this:

Development files for exiv2.

Distro packaging links:

## Links to Distribution Packages

- Debian: https://packages.debian.org/sid/libexiv2-dev
  - REQUIRED
- Ubuntu: https://packages.ubuntu.com/focal-updates/libexiv2-dev
  - REQUIRED
- Fedora: https://packages.fedoraproject.org/pkgs/exiv2/exiv2-devel/
  - IF AVAILABLE
- Arch: https://archlinux.org/packages/extra/x86_64/exiv2/
  - IF AVAILABLE
- Gentoo: https://packages.gentoo.org/packages/media-gfx/exiv2
  - IF AVAILABLE
- macOS: https://formulae.brew.sh/formula/exiv2
  - IF AVAILABLE
- Alpine: https://pkgs.alpinelinux.org/package/edge/community/x86_64/exiv2-dev
  - IF AVAILABLE
- NixOS/nixpkgs: https://search.nixos.org/packages?channel=24.05&show=exiv2&from=0&size=50&sort=relevance&type=packages&query=exiv2
  - OPTIONAL
- openSUSE: N/A
  - IF AVAILABLE
- rhel: https://centos.pkgs.org/9-stream/centos-crb-x86_64/exiv2-devel-0.27.5-2.el9.x86_64.rpm.html
  - IF AVAILABLE

### CODE REVIEW

**Comment by wjwwood** [comment]
There's a lint warning that suggests an OpenSUSE name for it, can you check that?

**Comment by peci1** [comment]
Thanks for spotting it. Added.

**Review by sloretz — APPROVED** [review]

File: rosdep/base.yaml [hunk 1]
```diff
@@ -1023,6 +1023,19 @@ exiv2:
       packages: [exiv2]
   rhel: [exiv2]
   ubuntu: [exiv2]
+exiv2-dev:
```
Comment (clalancette) [inline rosdep/base.yaml:1026]: Our policy is to have the key name match the Ubuntu name, so this should be `libexiv2-dev`.

**Review by clalancette — CHANGES_REQUESTED** [review]

File: rosdep/base.yaml (hunk 1 above)
Comment (peci1) [reply rosdep/base.yaml]: Okay, renamed

**Review by clalancette — APPROVED** [review]
Thanks!  Looks good to me.

**Comment by clalancette** [comment]
This now needs a rebase to fix conflicts; once that is done it will be good to merge.

**Comment by peci1** [comment]
@clalancette I'm on phone only now so rebasing would be difficult. Could you squash merge? Also, CI timed out, can you rerun it?

**Pull request merged**